backend/tests/unit/simulation/parameters.txt

```diff
@@ -0,0 +1,29 @@
+Scenario: parameters services behaves properly
+Given: A PCAC parameter service
+When: parameter mode is not equal to pc_ac
+Then: parameters are not updated as per the parameter requests
+
+Scenario: parameters services behaves properly
+Given: A PCAC parameter service
+When: parameter mode is equal to pc_ac
+Then: parameters are updated as per the parameter requests
+
+Scenario: parameters services behaves properly
+Given: A PCAC parameter service
+When: parameter request fio2 is less than fio2_min
+Then: parameters are not updated as per the parameter requests
```

Comment:
We should make a stronger assertion: the parameter remains at the previous value it had before the request was passed in. An alternative strategy would have been to update the parameter differently than requested, e.g. if the requested value was lower than the min allowed value, we would've just updated to the min allowed value instead (which is not as per the request). I made a decision here to instead ignore the requested value entirely, and we need a test to express this explicitly so that my design decision can be reviewed by other people.